modules/bim/spec/support/pages/ifc_models/show_default.rb

```diff
@@ -38,7 +38,7 @@ def initialize(project)
       end
 
       def path
-        defaults_ifc_models_project_ifc_models_path(project)
```

Comment:
This path is still used at two other places. Because of that the url `projects/foo/ifc_models` (e.g when clicking on "Manage Models") returns an error.